indico/htdocs/js/indico/jquery/itempicker.js

```diff
@@ -86,6 +89,16 @@
             self.element.qtip('destroy');
         },
 
+        _createItemsDict: function(items) {
+            var itemDict = {};
+
+            $.each(items, function(index, itemData) {
+                itemDict[itemData['id']] = {itemData: itemData, jQueryItem: null};
```

Comment:
`itemData.id`, and for the keys I'd rather go for something like `item`/`data` and `elem`